README.md

```diff
@@ -20,44 +20,48 @@
   Bearer offers a free, open solution, Bearer CLI, and a commercial solution, Bearer Pro, available through <a href="https://cycode.com/">Cycode</a>.
   <br /><br />
 
-  [Getting Started](#rocket-getting-started) - [FAQ](#question-faqs) - [Documentation](https://docs.bearer.com) - [Report a Bug](https://github.com/Bearer/bearer/issues/new/choose) - [Discord Community][discord]
+[Getting Started](#rocket-getting-started) - [FAQ](#question-faqs) - [Documentation](https://docs.bearer.com) - [Report a Bug](https://github.com/Bearer/bearer/issues/new/choose) - [Discord Community][discord]
 
-  [![GitHub Release][release-img]][release]
-  [![Test][test-img]][test]
-  [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
-  [![Discord](https://img.shields.io/discord/1042147477765242973?label=discord)][discord]
-</div>
+[![GitHub Release][release-img]][release]
+[![Test][test-img]][test]
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
+[![Discord](https://img.shields.io/discord/1042147477765242973?label=discord)][discord]
 
+</div>
 
 ## Language Support
+
 **Bearer CLI**: Go • Java • JavaScript • TypeScript • PHP • Python • Ruby <br/>
-**Bearer Pro by Cycode**: C# • Kotlin • Elixir *+ everything in Bearer OSS*
+**Bearer Pro by Cycode**: Java _(Cross-file support)_ • C# • Kotlin • Elixir • VB.Net _+ everything in Bearer OSS_
 
 <a href="https://docs.bearer.com/reference/supported-languages/">Learn more about language suppport</a>
 
 ## Developer friendly static code analysis for security and privacy
+
 <hr/>
```

Comment:
This is existing code, but since we're updating the file, I think we can remove this to get rid of the double line

![Screenshot 2025-04-24 at 16 46 31](https://github.com/user-attachments/assets/7dccdb7a-478b-49d3-afb5-81ca7a7c4af8)

```suggestion
```